rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -800,12 +812,33 @@ def _maybe_decode_erc20_721_transfer(
                 event=transfer,
             ),
         )
-        return DecodingOutput(event=transfer, matched_counterparty=maybe_counterparty)
+        return DecodingOutput(
+            event=transfer,
+            matched_counterparty=enrichment_output.matched_counterparty,
+            refresh_balances=enrichment_output.refresh_balances,
+        )
+
+    def _post_process(self, refresh_balances: bool) -> None:
+        """
+        Method that handles actions that have to be taken after a batch of transactions gets
+        decoded. Currently may only send a websocket message to the frontend to refresh balances.
+
+        The caller of decode_transactions should call this method after a batch of transactions
+        has been decoded.
+        """
+        if refresh_balances is True:
+            self.msg_aggregator.add_message(
+                message_type=WSMessageType.REFRESH_BALANCES,
+                data={
+                    'section': 'blockchain_balances',
```

Comment:
Shouldn't this be type?

```suggestion
                    'type': 'blockchain_balances',
```